app/views/pages/components/author-profile/page-content.ejs

```diff
@@ -1,7 +1,14 @@
 <link rel="stylesheet" href="/stylesheets/author-profile.css">
 <link rel="stylesheet" href="/stylesheets/category-listing.css">
 <div class='author-profile'>
-  <img class='author-profile__image' src="/image/<%=user.link%>" />
+  <!-- <img class='author-profile__image' src="/image/<%=user.link%>" /> -->
+  <%- include(`../../../ssrUtils/responsiveImage`, {
+    url: `/image/${user.link}`,
+    imageSizes: [650, 400, 400],
```

Comment:
the numbers should be respectively defined for:
650px, 1024px, 1920px, 2048px
You've listed only 650, 1024 and 1920

Reply:
I guess that it should be `[650, 400, 400, 400]`

Reply:
Fixed